src/prx/rinex_nav/test/test_nav_file_discovery.py

```diff
@@ -41,7 +41,7 @@ def test_find_local_ephemeris_file(set_up_test):
     assert isinstance(aux_files, dict)
 
 
-def test_download_remote_ephemeris_files(set_up_test):
+def disabled_test_download_remote_ephemeris_files(set_up_test):
```

Comment:
FTP connection got refused by IGN server when running in github action environment, opened